classes/meta-box.php

```diff
@@ -55,12 +55,11 @@ public function register_hooks() {
 			add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_scripts' ] );
 		}
 
-		// Load the editor script when on an elementor edit page.
-		// phpcs:ignore WordPress.PHP.NoSilencedErrors.Discouraged -- This deprecation will be addressed later.
-		$get_action             = filter_input( INPUT_GET, 'action', @FILTER_SANITIZE_STRING );
-		$is_elementor_edit_page = $pagenow === 'post.php' && $get_action === 'elementor';
+		// Load the editor script when on an Elementor edit page.
+		$get_action             = htmlspecialchars( $_GET['action'], ENT_QUOTES, 'UTF-8' );
```

Comment:
To avoid WARNING of undefined `action` key when using `$_GET['action']`
```suggestion
		$get_action             = htmlspecialchars( $_GET['action'] ?? null, ENT_QUOTES, 'UTF-8' );
```